archlinuxcn/3dslicer-bin/lilac.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+from lilaclib import *
+import subprocess
+
+def pre_build():
+  update_pkgver_and_pkgrel(_G.newver.lstrip('v'))
+  cmd = "curl https://download.slicer.org/ |grep bitstream |cut -d '/' -f 3 |cut -d '\"' -f 1 |sed -n 3p"
```

Comment:
Try to use escape character `\'` for `'` here
`cmd = 'curl https://download.slicer.org/ |grep bitstream |cut -d \'/\' -f 3 |cut -d \'"\' -f 1 |sed -n 3p'`